CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Optimizely Java X SDK Changelog
 
+## [4.0.0-beta]
+May 5th, 2023
+
+### New Features 
+The 4.0.0-beta release introduces a new primary feature, [Advanced Audience Targeting]( https://docs.developers.optimizely.com/feature-experimentation/docs/optimizely-data-platform-advanced-audience-targeting) 
+enabled through integration with [Optimizely Data Platform (ODP)](https://docs.developers.optimizely.com/optimizely-data-platform/docs) (
+[#474](https://github.com/optimizely/java-sdk/pull/474),
+[#481](https://github.com/optimizely/java-sdk/pull/481),
+[#482](https://github.com/optimizely/java-sdk/pull/482),
+[#483](https://github.com/optimizely/java-sdk/pull/483),
+[#484](https://github.com/optimizely/java-sdk/pull/484),
+[#485](https://github.com/optimizely/java-sdk/pull/485),
+[#487](https://github.com/optimizely/java-sdk/pull/487),
+[#489](https://github.com/optimizely/java-sdk/pull/489),
+[#490](https://github.com/optimizely/java-sdk/pull/490),
+[#494](https://github.com/optimizely/java-sdk/pull/494),
+[#495](https://github.com/optimizely/java-sdk/pull/495),
+[#496](https://github.com/optimizely/java-sdk/pull/496),
+[#497](https://github.com/optimizely/java-sdk/pull/497),
+[#498](https://github.com/optimizely/java-sdk/pull/498),
+[#500](https://github.com/optimizely/java-sdk/pull/500),
+[#501](https://github.com/optimizely/java-sdk/pull/501),
+[#502](https://github.com/optimizely/java-sdk/pull/502),
+[#503](https://github.com/optimizely/java-sdk/pull/503),
+[#504](https://github.com/optimizely/java-sdk/pull/504),
+[#509](https://github.com/optimizely/java-sdk/pull/509),
+[#511](https://github.com/optimizely/java-sdk/pull/511),
+[#512](https://github.com/optimizely/java-sdk/pull/512),
+[#513](https://github.com/optimizely/java-sdk/pull/513),
+[#514](https://github.com/optimizely/java-sdk/pull/514)
```

Comment:
This list is too long. Can we remove fix PRs (495-514), so can focus on new feature additions.